### PR TITLE
Update profile

### PR DESCRIPTION
**Pull request by kristijanH1998**
update profile endpoint finished

### CODE REVIEW

File: src/controllers/profile.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    country,
+    first_name,
+    last_name,
+    profile_picture,
+    biography,
+  } = req.body;
+
+  const user = get(req, 'identity');
+
+  if (!user) return res.sendStatus(500);
```
Comment (JJeshua) [inline src/controllers/profile.js]: redundant, we already know the user exists from the isAuthenticated middleware

File: src/controllers/profile.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      profile_picture,
+      biography,
+    };
+
+    await updateUserProfile(user._id, update);
+
+    return res.sendStatus(200);
+  } catch (error) {
+    console.error('Error updating profile: ', error);
+    return res.sendStatus(400).json({
```
Comment (JJeshua) [inline src/controllers/profile.js]: should be res.status(500).json({...

**Review by vlomeli — APPROVED** [review]
LGTM

**Review by JJeshua — APPROVED** [review]
LGTM

**Pull request merged**